tests/tests_pytorch/trainer/logging_/test_eval_loop_logging.py

```diff
@@ -765,7 +765,7 @@ def test_dataloader(self):
             return [super().test_dataloader(), super().test_dataloader()]
 
     model = CustomBoringModel()
-    trainer = Trainer(default_root_dir=tmpdir, limit_test_batches=1)
+    trainer = Trainer(default_root_dir=tmpdir, limit_test_batches=1, logger=TensorBoardLogger(tmpdir))
```

Comment:
@awaelchli Should we also look into making `save_dir` None by default so that it can be replaced with `Trainer.default_root_dir` at setup time? (not in this PR)

Reply:
Not sure what you mean. It is already like this: If user does not set save dir in the logger, trainer will log to default root dir